lib/minitest/mock.rb

```diff
@@ -113,8 +114,11 @@ def verify
 
     def method_missing(sym, *args, &block) # :nodoc:
       unless @expected_calls.key?(sym) then
-        raise NoMethodError, "unmocked method %p, expected one of %p" %
-          [sym, @expected_calls.keys.sort_by(&:to_s)]
+        if @delegator
+          delegate_method(sym, *args, &block)
+        else
+          raise_unmocked_method_error(sym)
+        end
```

Comment:
What does this do? It looks like it calls the method if there is a delegate, and promptly drops the results on the floor and goes on to raise an exception because the method called wasn't mocked. If there is no delegator, it raises an exception. Why?